dnf-behave-tests/features/plugins-core/versionlock-lock.feature

```diff
@@ -96,6 +96,20 @@ Scenario: I can remove the installed locked version of the package
         | remove        | wget-0:1.19.5-5.fc29.x86_64           |
 
 
+@bz1780370
```

Comment:
Shouldn't you rather tag with the original bug instead of the new one? (and are you going to mark the new bug as a duplicate?)

Reply:
The new bug is not strictly a duplicate - although they have the same root cause the bugs refer to different commands - downgrade / remove.